src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -505,6 +506,7 @@ func (c controller) sendCompleteInstallation(isSuccess bool, errorInfo string) {
  **/
 func (c controller) uploadSummaryLogs(podName string, namespace string, sinceSeconds int64, isMustGatherEnabled bool) error {
 	var tarentries = make([]utils.TarEntry, 0)
+	var ok bool = true
```

Comment:
maybe better to change if lines :
tarfile, err := c.collectMustGatherLogs(ctx)
if err == nil {
			if entry, tarerr := utils.NewTarEntryFromFile(tarfile); tarerr == nil {
				tarentries = append(tarentries, *entry)
			}
		}
and then to return err and will be no need to have this bool?

Reply:
No. I wanted the flow to be forgiving making the most of the logs collection and not stopping if one of the component fails. However, if one of them fails, I'd like to retry and therefore at the end, I return error to activate the retry mechanism

Reply:
this is what i meant, i just think this var is not relevant , just take err and return it at the end

Reply:
agree with u